deployment/README.md

```diff
@@ -0,0 +1,252 @@
+| TF_state_bucket                |  create-k8s (aws) | the s3 bucket to store the deployed terraform state                                               |
+| TF_state_bucket_key            |  create-k8s (aws) | the s3 key to save the deployed terraform state.tf                                                |
+| TF_VAR_vpc_cidr_block          |  create-k8s (aws) | AWS vpc cidr block                                                                                |
+| TF_VAR_azs                     |  create-k8s (aws) | A list of regional availability zones for the AWS vpc subnets                                     |
+| TF_VAR_public_subnets          |  create-k8s (aws) | A list of cidr blocks for AWS public subnets                                                      |
+| TF_VAR_private_subnets         |  create-k8s (aws) | A list of cidr blocks for AWS private subnets                                                     | 
+| TF_VAR_eks_cluster_name        |  create-k8s (aws) | EKS cluster name                                                                                  |
+| TF_VAR_eks_cluster_version     |  create-k8s (aws) | EKS cluster version, possible options: 1.18.16, 1.19.8, 1.20.7, 1.21.2                            |
+| TF_VAR_eks_node_groupname      |  create-k8s (aws) | EKS worker node group name                                                                        |
+| TF_VAR_eks_node_ami_type       |  create-k8s (aws) | EKS worker node group AMI type, possible options: AL2_x86_64, AL2_x86_64_GPU, AL2_ARM_64, CUSTOM  | 
+| TF_VAR_eks_node_disk_size      |  create-k8s (aws) | disk size (GiB) for EKS worker nodes                                                              |
+| TF_VAR_eks_node_instance_types |  create-k8s (aws) | A list of instance types for the EKS worker nodes                                                 |
+| TF_VAR_eks_node_min_size       |  create-k8s (aws) | minimum number of eks worker nodes                                                                |
+| TF_VAR_eks_node_desired_size   |  create-k8s (aws) | desired number of eks worker nodes                                                                |
+| TF_VAR_eks_node_max_size       |  create-k8s (aws) | maximum number of eks worker nodes                                                                |
+| TF_VAR_eks_capacity_type       |  create-k8s (aws) | type of capacity associated with the eks node group, possible options: ON_DEMAND, SPOT            |
+| TF_VAR_ec2_ssh_key             |  create-k8s (aws) | ec2 key Pair name for ssh access (must create this key pair in your AWS account before)           |
+
+Notes:
+
+- create-k8s refers to the [create-k8s.sh][create-k8s-sh] script
+
+- fuel-core-deploy refers to [fuel-core-deploy][fuel-deploy-script] script
+
+- for 'base64_github_auth_token' environment variable:
```

Comment:
I'm thinking we should strip out all the github regcred setup. It creates unnecessary risk to users by storing a PAT in the k8s cluster. Instead, we should maybe look into publishing our images to a public registry such as DockerHub in a similar fashion to how we retag images to ECR.

Reply:
Actually now that fuel-core is public, it seems like I can do commands like `docker pull ghcr.io/fuellabs/fuel-core:v0.3.2` without needing to login to the registry. So maybe this will just work already?

Reply:
@Voxelot - your point is correct and I adjusted with the last code update.